test/amoy.test.ts

```diff
@@ -0,0 +1,29 @@
+import { expect } from "chai"
+import { fetchFirehoseBlock } from "./lib/firehose"
+import { isNetwork } from "./lib/network"
+
+if (isNetwork("amoy")) {
+  describe("Amoy", function (){
```

Comment:
Do we have `Polygon` tests already? Would probably be good to actually merge it under Polygon?